app/components/search/Header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
```

Comment:
This component was extracted from the searchResultsPage and serviceDiscoveryResults page below. The only new changes involve the addition of the QR code modal